src/Land/Model/SoilHeatParameterizations.jl

```diff
@@ -0,0 +1,196 @@
+"""
+    SoilHeatParameterizations
+
+Functions for volumetric heat capacity, temperature as a function
+of volumetric internal energy, saturated thermal conductivity, thermal
+conductivity, the Kersten number, and relative saturation are included.
+"""
+module SoilHeatParameterizations
+
+using DocStringExtensions
+
+export volumetric_heat_capacity,
+    internal_energy,
+    saturated_thermal_conductivity,
+    thermal_conductivity,
+    relative_saturation,
+    kersten_number,
+    volumetric_internal_energy,
+    temperature_from_I
+
+
+"""
+    function temperature_from_I(
+        T_ref::FT,
+        I::FT,
+        θ_i::FT,
+        ρ_i::FT,
+        LH_f_0::FT,
+        cs::FT
+    ) where {FT}
+
+Computes the temperature of soil given `θ_i` and volumetric internal energy `I`.
+"""
+function temperature_from_I(
+    T_ref::FT,
+    I::FT,
+    θ_i::FT,
+    ρ_i::FT,
+    LH_f0::FT,
+    cs::FT,
+) where {FT}
+    T = T_ref + (I + θ_i * ρ_i * LH_f0) / cs
+    return T
+end
+
+"""
+    volumetric_heat_capacity(
+        θ_l::FT,
+        θ_i::FT,
+        c_ds::FT,
+        cp_l::FT,
+        cp_i::FT
+    ) where {FT}
+
+Compute the expression for volumetric heat capacity.
+"""
+function volumetric_heat_capacity(
+    θ_l::FT,
+    θ_i::FT,
+    c_ds::FT,
+    cp_l::FT,
+    cp_i::FT,
+) where {FT}
+    c_s = c_ds + θ_l * cp_l + θ_i * cp_i
+    return c_s
+end
```

Comment:
```suggestion
function volumetric_heat_capacity(
    θ_l::FT,
    θ_i::FT,
    c_ds::FT,
    param_set::AbstractParameterSet,
) where {FT}
    _ρ_i = FT(ρ_cloud_ice(param_set))
    ρ_i_cp_i = FT(cp_i(param_set) * _ρ_i)

    _ρ_l = FT(ρ_cloud_liq(param_set))
    ρ_l_cp_l = FT(cp_l(param_set) * _ρ_l)
    c_s = c_ds + θ_l * ρ_l_cp_l + θ_i * ρ_i_cp_i
    return c_s
end
```
Also, why

Reply:
A lot of these functions require the same parameters from param_set and are called at the same time. So we have a single function which gets all the params, and then each of these functions takes them as arguments

Reply:
Sounds good, Ill add in param_set, and remove the function get_clima_params_for_heat